area/HighTowerOfSorcery.are

```diff
@@ -4820,7 +4820,7 @@ a scroll written on blue paper~
 A blue scroll has been left here.~
 vellum~
 scroll G A
-35 24 1 1 0
+35 24 0 0 0
```

Comment:
We had a few scrolls of enchant weapon paired with armour.

Reply:
Nice spot!